setup.py

```diff
@@ -37,6 +38,10 @@
 from tfx.tools import resolve_deps
 from wheel import bdist_wheel
 
+# Prefer to import `package_config` from the setup.py script's directory.
+sys.path.insert(0, os.path.dirname(__file__))
+import package_config
```

Comment:
Add some explanation on how this selected during wheel building?

Reply:
Done

Reply:
Done.